tools/benchmark/cmd/watch_latency.go

```diff
@@ -40,15 +39,22 @@ var watchLatencyCmd = &cobra.Command{
 }
 
 var (
-	watchLTotal     int
-	watchLPutRate   int
-	watchLKeySize   int
-	watchLValueSize int
+	watchLPutTotal       int
+	watchLPutRate        int
+	watchLKeySize        int
+	watchLValueSize      int
+	watchLStreams        int
+	watchLWatchPerStream int
+	watchLPrevKV         bool
 )
 
 func init() {
 	RootCmd.AddCommand(watchLatencyCmd)
-	watchLatencyCmd.Flags().IntVar(&watchLTotal, "total", 10000, "Total number of put requests")
+	watchLatencyCmd.Flags().IntVar(&watchLStreams, "streams", 10, "Total watch streams")
+	watchLatencyCmd.Flags().IntVar(&watchLWatchPerStream, "watch-per-stream", 10, "Total watchers per stream")
+	watchLatencyCmd.Flags().BoolVar(&watchLPrevKV, "prevkv", false, "PrevKV enabled on watch requests")
+
+	watchLatencyCmd.Flags().IntVar(&watchLPutTotal, "put-total", 1000, "Total number of put requests")
```

Comment:
Should we be trying to align the defaults across the board at `10,000`?

```bash
 james  ~  D  e  t  benchmark  ?6  benchmark-put-..                                                                                  19:30:38 
 ➜ grep -Ri "1000" | grep -i intvar
cmd/put.go:     putCmd.Flags().IntVar(&putTotal, "total", 10000, "Total number of put requests")
cmd/stm.go:     stmCmd.Flags().IntVar(&stmTotal, "total", 10000, "Total number of completed STM transactions")
cmd/txn_put.go: txnPutCmd.Flags().IntVar(&txnPutTotal, "total", 10000, "Total number of txn requests")
cmd/watch.go:   watchCmd.Flags().IntVar(&watchPutTotal, "put-total", 1000, "Number of put requests")
cmd/range.go:   rangeCmd.Flags().IntVar(&rangeTotal, "total", 10000, "Total number of range requests")
cmd/watch_get.go:       watchGetCmd.Flags().IntVar(&watchGetTotalWatchers, "watchers", 10000, "Total number of watchers")
cmd/lease.go:   leaseKeepaliveCmd.Flags().IntVar(&leaseKeepaliveTotal, "total", 10000, "Total number of lease keepalive requests")
cmd/watch_latency.go:   watchLatencyCmd.Flags().IntVar(&watchLPutTotal, "put-total", 1000, "Total number of put requests")
cmd/mvcc.go:    mvccCmd.PersistentFlags().IntVar(&batchLimit, "batch-limit", 10000, "A limit of batched transaction")
cmd/txn_mixed.go:       mixedTxnCmd.Flags().IntVar(&mixedTxnTotal, "total", 10000, "Total number of txn requests")

```

Reply:
Benchmark code is not very organized. I planned to make a direct improvement that we require for testing prevKV, and leave cleanups for later.